topics/react-router/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: react-router, react router
+created_by: Ryan Florence & Michael Jackson
+display_name: react-router
```

Comment:
The display name is usually the way the name looks in contexts where it's being written about as a proper noun.
```suggestion
display_name: React Router
```